testing/src/main/java/org/oppia/android/testing/RecyclerViewMatcher.kt

```diff
@@ -1,83 +1,100 @@
+            )
           }
-          description.appendText("with id: $idDescription")
         }
+        description.appendText("with id: $idDescription")
+      }
 
-        public override fun matchesSafely(view: View): Boolean {
-          this.resources = view.resources
-          if (childView == null) {
-            val recyclerView = view.rootView.findViewById<View>(recyclerViewId) as RecyclerView
-            if (recyclerView.id == recyclerViewId) {
-              childView = recyclerView.findViewHolderForAdapterPosition(position)!!.itemView
-            } else {
-              return false
-            }
-          }
-          return if (targetViewId == -1) {
-            view === childView
+      public override fun matchesSafely(view: View): Boolean {
+        this.resources = view.resources
+        if (childView == null) {
+          val recyclerView = view.rootView.findViewById<View>(recyclerViewId) as RecyclerView
+          if (recyclerView.id == recyclerViewId) {
+            childView = recyclerView.findViewHolderForAdapterPosition(position)!!.itemView
           } else {
-            val targetView = childView!!.findViewById<View>(targetViewId)
-            view === targetView
+            return false
           }
         }
+        return if (targetViewId == -1) {
+          view === childView
+        } else {
+          val targetView = childView!!.findViewById<View>(targetViewId)
+          view === targetView
+        }
       }
     }
+  }
 
-    /** Returns item count ViewAssertion for a recycler view. */
-    fun hasItemCount(count: Int): ViewAssertion {
-      return RecyclerViewItemCountAssertion(count)
-    }
+  /** Returns item count ViewAssertion for a recycler view. */
+  fun hasItemCount(count: Int): ViewAssertion {
+    return RecyclerViewItemCountAssertion(count)
+  }
 
-    /** Returns span count ViewAssertion for a recycler view that use GridLayoutManager. */
-    fun hasGridItemCount(spanCount: Int, position: Int): ViewAssertion {
-      return RecyclerViewGridItemCountAssertion(spanCount, position)
-    }
+  /** Returns span count ViewAssertion for a recycler view that use GridLayoutManager. */
+  fun hasGridItemCount(spanCount: Int, position: Int): ViewAssertion {
+    return RecyclerViewGridItemCountAssertion(spanCount, position)
+  }
 
-    fun hasGridColumnCount(expectedColumnCount: Int): ViewAssertion {
-      return GridLayoutManagerColumnCountAssertion(expectedColumnCount)
-    }
+  fun hasGridColumnCount(expectedColumnCount: Int): ViewAssertion {
+    return GridLayoutManagerColumnCountAssertion(expectedColumnCount)
+  }
+
+  fun verifyItemDisplayedOnRecyclerView(
+    recyclerView: Int,
+    itemPosition: Int,
+    targetView: Int
+  ) {
+    Espresso.onView(
+      atPositionOnView(
+        recyclerViewId = recyclerView,
+        position = itemPosition,
+        targetViewId = targetView
+      )
+    ).check(ViewAssertions.matches(ViewMatchers.isDisplayed()))
```

Comment:
import `matches` not `ViewAssertions`.

Same with `ViewMatchers`

Reply:
Done